tests/ui/swap.rs

```diff
@@ -54,7 +50,7 @@ fn main() {
     a = b;
     b = a;
 
-    ; let t = a;
+;    let t = a;
```

Comment:
I'm not sure if this change is desired. Is it a bug in rustfmt?

Reply:
Probably, I already filed a ticket for this (https://github.com/rust-lang/rustfmt/issues/3245), can `[rustfmt::skip]` later.